ovos_plugin_common_play/ocp/gui.py

```diff
@@ -210,21 +210,32 @@ def manage_display(self, page_requested, timeout=None):
                 self.show_page(self.disambiguation_playlists_page, override_idle=timeout, override_animations=True)
             else:
                 self.show_page(self.disambiguation_playlists_page, override_idle=True, override_animations=True)
-                
+
+        if self.player.enable_app_view_timeout and page_requested == "player":
+            self.player.event_scheduler_interface.schedule_event(
+                self.timeout_app_view, 30, data=None, name="ocp_app_view_timer")
```

Comment:
30 should be self.player.app_view_timeout, maybe move this into a helper method as suggested in the other comment